waitress/tests/test_adjustments.py

```diff
@@ -219,6 +219,17 @@ def test_ipv4_disabled(self):
     def test_ipv6_disabled(self):
         self.assertRaises(ValueError, self._makeOne, ipv6=False, listen="[::]:8080")
 
+    def test_server_header_removable(self):
+        inst = self._makeOne(ident=None)
+        self.assertEqual(inst.ident, None)
+
+        inst = self._makeOne(ident='')
+        self.assertEqual(inst.ident, '')
```

Comment:
In this case the `ident` should be `None`. So that if a user sets it to an empty string in a config file, there is no ident sent.